src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -12,7 +12,7 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+    public static final String MESSAGE_SUCCESS = "All contacts have been cleared!";
```

Comment:
TA2 has been cleared?

Reply:
Thought about that too, but I think "contacts" may be clearer. Since you are essentially just deleting all persons

Reply:
person and assignments are whole part relationship?

Reply:
yep. Every person has his/her own assignment list. if that person is deleted, the assignments are similarly deleted as well